cpp/example_code/s3/get_put_bucket_acl.cpp

```diff
@@ -272,15 +272,21 @@ int main()
     Aws::SDKOptions options;
     Aws::InitAPI(options);
     {
-        const Aws::String bucket_name = "my-bucket";
-
-        // Set the ACL's owner information. 
+        //TODO: Change to the name of an actual bucket in your account.
+        //Else, you will get either an Access Denied or a NoSuchBucket error (depending on whether the bucket exists elsewhere).
+        const Aws::String bucket_name = "DOC-EXAMPLE-BUCKET";
+        //TODO: Set to the region of your profile.
+        const Aws::String region = "us-east-1";
+
+        //TODO: Set the ACL's owner information (if it is your bucket, then you want your canonical id). 
+        //See https://docs.aws.amazon.com/AmazonS3/latest/userguide/finding-canonical-user-id.html for more information.
```

Comment:
Okay, I see you tell them where to find their canonical ID. Add this comment to the previous place where I made my comment.